ios/Classes/SwiftSoundStreamPlugin.swift

```diff
@@ -195,7 +195,7 @@ public class SwiftSoundStreamPlugin: NSObject, FlutterPlugin {
     private func resetEngineForRecord() {
         mAudioEngine.inputNode.removeTap(onBus: mRecordBus)
         let input = mAudioEngine.inputNode
-        let inputFormat = input.outputFormat(forBus: mRecordBus)
+        let inputFormat = input.inputFormat(forBus: mRecordBus)
```

Comment:
I'm wondering: are we suppose to get what the node outputs?
It's like:
`m_input -> micNode -> m_output -> c_input -> converter -> c_output`

Reply:
I think for input nodes, input* are all we want.

https://developer.apple.com/documentation/avfaudio/avaudionode

inputNode/outputNode share the same grandparent, AudioNode. I guess AVAudioInputNode overrides inputFormat, input*, etc and AVAudioOutputNode overrides outputFormat, output*, etc.

Anyway, working on a version to use AVAudioSinkNode instead of installTap. Frame size can be tuned and low-latency recording logic can be achieved, but requires iOS 13.0.